README.md

```diff
@@ -107,6 +108,30 @@ const logger = loggerFactory.init({
 
 ```
 
+### multi streams
+
+To be compatible with previous versions. When there is `logStream` key in config, multi streams config key `logStreams` would be dismissed.
+
+Using `logStreams` config key to set multi streams when `loggerFactory.init`.
+
+```js
+const logger = loggerFactory.init({
+  logName: 'multiStream',
+  logStreams: [
+    {
+      logStream: 'stdout'
+    }, 
+    {
+      logName: 'multiStream',
```

Comment:
is `logName` part of the logStream?

Reply:
Only when `logProto === 'sys'`, this value would be used in `bunyan-syslog`:

bunyan-logger-factory::index.js::getStream::bsyslog.createBunyanStream::param.type

```js
const getStream = (config) => {
  let stream;
  switch (config.streamType.toUpperCase()) {
    case 'FILE':
      stream = { path: config.streamPath || './logs/app.log' };
      break;
    case 'SYSLOG':
      stream = {
        type: 'raw',
        stream: bsyslog.createBunyanStream({
          // ...
          type: config.logProto,
```

=> bunyan-syslog/lib/index.js::createBunyanStream, case `ops.type === sys` (from our config logProto)

```js
module.exports = {
        createBunyanStream: function createBunyanStream(opts) {
                assert.optionalObject(opts, 'options');
                opts = opts || {};

                var type;

                switch (opts.type) {
                case 'sys':
                        type = SyslogStream;
                        break;
```

=> bunyan-syslog/lib/sys.js, `this.name = opts.name || process.title || process.argv[0];`, and finally `binding.openlog(this.name, binding.LOG_CONS, 0);`

```js
function SyslogStream(opts) {
        assert.object(opts, 'options');
        assert.optionalNumber(opts.facility, 'options.facility');
        assert.optionalString(opts.name, 'options.name');

        Stream.call(this);

        this.facility = opts.facility || 1;
        this.name = opts.name || process.title || process.argv[0];
        this.writable = true;

        if (this.constructor.name === 'SyslogStream') {
                binding.openlog(this.name, binding.LOG_CONS, 0);
                process.nextTick(this.emit.bind(this, 'connect'));
        }
}
```

Reply:
As I talked with kaleo, the logProto we using is `tcp`, so this `logName` actually is never used.

Reply:
ok, you can keep it as is, just it's a bit confusing.